components/org.wso2.carbon.identity.api.server.common/src/main/java/org/wso2/carbon/identity/api/server/common/error/bulk/BulkErrorResponse.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.server.common.error.bulk;
+
+import org.apache.commons.logging.Log;
+import org.wso2.carbon.identity.api.server.common.error.ErrorDTO;
+
+import java.util.List;
+
+import static org.wso2.carbon.identity.api.server.common.Util.getCorrelation;
+import static org.wso2.carbon.identity.api.server.common.Util.isCorrelationIDPresent;
+
+/**
+ * Common BulkErrorResponse Object for all the server bulk API related errors.
+ */
+public class BulkErrorResponse extends BulkErrorDTO {
+
+    private static final long serialVersionUID = 373229260532218871L;
+
+    /**
+     * BulkErrorResponse Builder.
+     */
+    public static class Builder {
+        private String code;
+        private String message;
+        private String description;
+        private List<? extends ErrorDTO> failedOperations;
+
+        public Builder() {
```

Comment:
Since you are not doing anything here, default constructor can handle it, right?

Reply:
Yes. Removed the duplicate constructor.